CONTRIBUTING.md

```diff
@@ -18,7 +18,7 @@ You can edit existing files or create new files directly via the [Github web int
 
 #### Fork the Project
 
-If you don't want to use the [Github web interface](https://github.com/GSA/search-gov/tree/master/), you can use the command line tools to fork the [project on Github](https://github.com/GSA/search-gov) and check out your copy.
+If you don't want to use the [Github web interface](./fork), you can use the command line tools to fork the [project on Github](https://github.com/GSA/search-gov) and check out your copy.
```

Comment:
Aaand it turns out that this can't be a relative link, because the relative links include the commit SHA, which isn't relevant to the `/fork` page. So this link should be an absolute link to https://github.com/GSA/search-gov/fork.

Reply:
Yup -- made this change.